### ISSUE TITLE
Compilation failure with recent MIME::Base32

### ISSUE THREAD

**Issue by eserte** [Songmu/p5-Riji#19]
With most recent MIME::Base32 the test suite of Riji fails. I see errors like:
```
Could not load class (Riji::Model::Blog) because : "RFC" is not exported by the MIME::Base32 module
```

**Comment by eserte** [comment]
See also https://rt.cpan.org/Ticket/Display.html?id=113838

**Comment by Songmu** [comment]
Thank you for the report! It caused by incompatible change of MIME::Base32 as you said.

Riji now depends on MIME::Base32 1.301 or later and fixed.

**Comment by Songmu** [comment]
released as Riji v0.9.8.

**Comment by Daniel-Peder** [comment]
hi, use the module without the RFC parameter.

eg:

use MIME::Base32;

it reportedly works:

2016-06-17 0:24 GMT+02:00 Paierl, Martin <martin.paierl@spaixx.ch>:
> Hi Daniel
>
> Sensational... this was the solution.
> Thank you very much...
> I can't believe getting such a fast reply in the middle of the night...
>
> I will write the maintainer...
>
> Best regards, Martin
> Again, thank you very, very much.
>
>
>
> -----Ursprüngliche Nachricht-----
> Von: daniel.peder@gmail.com [mailto:daniel.peder@gmail.com] Im Auftrag von Daniel Péder - www.infoset.cz
> Gesendet: Freitag, 17. Juni 2016 00:16
> An: Paierl, Martin <martin.paierl@spaixx.ch>
> Betreff: Re: "RFC" is not exported by the MIME::Base32 module
>
> .. eg try this:
>
> use MIME::Base32;
>
> and please contact the current maintainer of the package to correct
> the man content.
>
> 2016-06-17 0:09 GMT+02:00 Paierl, Martin <martin.paierl@spaixx.ch>:
>> Dear Daniel
>>
>> Please excuse coming directly to you.
>> I have an issue - and my knowledge to solve this problem is not enough good unfortunately.
>>
>> It is about this CPAN module:
>>
>> use MIME::Base32 qw( RFC );
>>
>> It causes at the moment:   "RFC" is not exported by the MIME::Base32 module....
>>
>> I found in the internet that I am not the only one to have this problem (cf. https://github.com/Songmu/p5-Riji/issues/19 )
>>
